tools/make-npm.sh

```diff
@@ -35,10 +35,11 @@ node -pe "JSON.stringify(fs.readFileSync('$THIRDPARTY/easyprivacy.txt', 'utf8'))
     > $DES/data/easyprivacy.json
 
 # Target-specific
-cp    platform/npm/*.json  $DES/
-cp    platform/npm/.*.json $DES/
-cp    platform/npm/*.js    $DES/
-cp -R platform/npm/tests   $DES/
+cp    platform/npm/.npmignore  $DES/
```

Comment:
Sorry, I have force-pushed a couple of times. There is a `.npmignore` file now and it ignores `results.json` and `tests/`.